### DISCUSSION TITLE
Real checkboxes, <input type="checkbox" instead of icons for visualization.

### DISCUSSION THREAD

**Discussion by jjandrorr** [vakata/jstree#2469] [Q&A]
Hello,

Im using jsTree - v3.3.8 configured with real_checkboxes.

For the visualization i would like to show instead the icons, the design corresponding with a real checkbox: <input type="checkbox" in order to keep the same aspect of other checkboxes used in other parts of the app. 

Is there any way to do this?

I have configured the tree with real_checboxes but looking in to the doom i don´t see any  input label.

Thanks in advance.

**Comment by vakata** [comment]
Take a look at the real checkboxes implementation (https://github.com/vakata/jstree/blob/master/src/misc.js#L75) and customize it as your own plugin - this is the only way to customize the markup.
